lib/bindings.ts

```diff
@@ -0,0 +1,110 @@
+import {randomBytes} from "node:crypto";
```

Comment:
was this file manually created? how was this working before?

Reply:
This `prepareBindings` was in `index.js`.  Made it a TS file for type safety and so can use the `prepareBindings` function in both the ESM and CJS index